src/Driver.cpp

```diff
@@ -30,13 +30,13 @@ ProbeMeasurements Driver::getMeasurements () {
     measurements.dissolved_oxygen_mgL = readSingleRegister<float>(R_DISSOLVED_OXIGEN_MG_L) / 100 * 1000;
     measurements.dissolved_oxygen_sat = readSingleRegister<float>(R_DISSOLVED_OXIGEN_SAT) / 100;
     measurements.temperature = base::Temperature::fromCelsius(readSingleRegister<float>(R_TEMPERATURE) / 100);
-    measurements.ph = readSingleRegister<float>(R_PH);
-    measurements.oxidation_reduction_potencial = readSingleRegister<float>(R_OXIDATION_REDUCTION_POTENCIAL) * (10^6);
-    measurements.conductivity = readSingleRegister<float>(R_CONDUCTIVITY) * (10^6);
-    measurements.salinity = readSingleRegister<float>(R_SALINITY);
-    measurements.dissolved_solids = readSingleRegister<float>(R_DISSOLVED_SOLIDS);
+    measurements.ph = readSingleRegister<float>(R_PH) / 100;
+    measurements.conductivity = readSingleRegister<float>(R_CONDUCTIVITY) / 100 * (10^6);
```

Comment:
Vício de ruby/python usando `^` para potencia.

```suggestion
    measurements.conductivity = readSingleRegister<float>(R_CONDUCTIVITY) / 100 * std::pow(10, 6);
```
ou
```
constexpr float scale = std::pow(10,6) / 100.0;
measurements.conductivity = readSingleRegister<float>(R_CONDUCTIVITY) * scale;
```